robottelo/host_helpers/cli_factory.py

```diff
@@ -717,15 +717,6 @@ def _setup_org_for_a_rh_repo(self, options=None):
             env_id = self.make_lifecycle_environment({'organization-id': org_id})['id']
         else:
             env_id = options['lifecycle-environment-id']
-        # Clone manifest and upload it
```

Comment:
One thing to consider: This removal means that the Org provided via `options` must have some manifest already added, otherwise it would fail on line 731/722 I guess. Same situation would happen when no Org is provided - new one is created on line 713, but no manifest is added.
To address this let's make the Org param mandatory OR upload some manifest here like we do [in fixtures](https://github.com/SatelliteQE/robottelo/blob/83ca28419c129aad304d49c89f758af36b9a74b7/pytest_fixtures/component/taxonomy.py#L102).

Whatever path you choose, please update the docstring too (L694).

Reply:
`self._satellite.cli.Subscription.exists({'organization-id': org_id})`  will return dictionary if manifest already uploaded, otherwise it will return empty dictionary